install.py

```diff
@@ -619,6 +619,7 @@ def driver():
         nargs="?",
         type=int,
         required=False,
+        default=os.environ.get("THREAD_COUNT"),
```

Comment:
Could we use conda build's built-in [`CPU_COUNT`](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process) envvar instead of defining our own?

Reply:
Unfortunately, the machines on NGC detect all the physical CPUs. The way that NGC CPU count limitation works is through limiting time share, not by hiding the actual physical GPUs. So we have to force the thread count to override what is detected. It would be OK to oversubscribe in general, but the machine also comes with memory limitation, causing the build to run out of memory if too many threads are requested.

Reply:
I renamed the variable as you suggested.